manifests/base/deployment.yaml

```diff
@@ -27,17 +27,17 @@ spec:
       serviceAccount: odh-dashboard
       containers:
       - name: odh-dashboard
-        image: odh-dashboard    # Replaced by the kustomize images parameter
+        image: odh-dashboard
         imagePullPolicy: Always
         ports:
         - containerPort: 8080
         resources:
           requests:
-            cpu: 300m
-            memory: 500Mi
-          limits:
             cpu: 500m
             memory: 1Gi
+          limits:
```

Comment:
Will anyone care that this went up?

Reply:
This was added upstream & downstream to minimize occurrence of OOMKill scenarios

Reply:
Sounds good.  Thanks